dialogflow-cx/vpc-sc-demo/frontend/src/PrivacyPolicy.js

```diff
@@ -0,0 +1,107 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
+
+function PrivacyPolicyPage(props) {
+  const openSourceAndPubliclyViewable = (
+    <Link
+      target="_blank"
+      href="https://developers.google.com/identity/protocols/oauth2"
```

Comment:
I think these two links are switched? Link for `openSourceAndPubliclyViewable` should be swapped with the link for `AccessToken`?

Reply:
Done. I set up an email alias in Cloud Domains.

Reply:
Eagle Eyes! Done.